src/inverseRobot.js

```diff
@@ -24,7 +24,20 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+  const robotKeys = Object.values(robot).reverse();
+  const robotValues = Object.keys(robot).reverse();
```

Comment:
Order of properties in resulting object doesn't matter at all. `reverse()` is useless.